src/routes/vocabulary-detail/components/AlternativeWordsSection.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import { widthPercentageToDP as wp } from 'react-native-responsive-screen'
+import styled from 'styled-components/native'
+
+import { ArrowRightIcon, CrystalBallIcon } from '../../../../assets/images'
+import { ContentSecondary, ContentTextBold } from '../../../components/text/Content'
+import { Document } from '../../../constants/endpoints'
+import labels from '../../../constants/labels.json'
+import theme from '../../../constants/theme'
+
+const Heading = styled(ContentSecondary)`
+  padding: ${props => props.theme.spacings.xs} 0;
+`
+
+const AlternativeWords = styled(ContentSecondary)`
+  margin-bottom: ${props => props.theme.spacings.md};
+`
+
+const Root = styled.View`
+  flex-direction: row;
+`
+
+const Content = styled.View`
+  padding: 0 10px;
```

Comment:
even there are 2px missing i think we should stick to our spacing classes
```suggestion
 padding: 0 ${props => props.theme.spacings.xs};
```